x/concentrated-liquidity/internal/math/math.go

```diff
@@ -96,22 +96,15 @@ func CalcAmount1Delta(liq, sqrtPriceA, sqrtPriceB sdk.Dec, roundUp bool) sdk.Dec
 
 // getNextSqrtPriceFromAmount0RoundingUp utilizes the current squareRootPrice, liquidity of denom0, and amount of denom0 that still needs
 // to be swapped in order to determine the next squareRootPrice
-// TODO: make an issue to determine if we can remove the less precise formula
+// Note: we are using only using the precise formula here.
```

Comment:
I think this comment should be removed, as no one will get the context months later